modules/auxiliary/scanner/http/grandstream_ucm62xx_sql_account_guess.rb

```diff
@@ -0,0 +1,219 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+# http://firmware.grandstream.com/Release_UCM62xx_1.0.20.22.zip
+
+class MetasploitModule < Msf::Auxiliary
+
+  include Msf::Exploit::Remote::HttpClient
+  include Rex::Proto::Http::WebSocket
+  include Msf::Auxiliary::Scanner
+  include Msf::Auxiliary::Report
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Grandstream UCM62xx IP PBX WebSocket Blind SQL Injection Credential Dump',
+        'Description' => %q{
+          This module uses a blind SQL injection (CVE-2020-5724) affecting the Grandstream UCM62xx
+          IP PBX to dump the users table. The injection occurs over a websocket at the websockify
+          endpoint, and specifically occurs when the user requests the challenge (as part of a
+          challenge and response authentication scheme). The injection is blind, but the server
+          response contains a different status code if the query was successful. As such, the
+          attack can guess the contents of the user database. Most helpfully, the passwords are
+          stored in cleartext within the user table (CVE-2020-5723).
+
+          This issue was patched in Grandstream UCM62xx IP PBX firmware version 1.20.22.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [
+          'jbaines-r7' # Vulnerability discovery, original poc, and Metasploit module
+        ],
+        'References' => [
+          [ 'CVE', '2020-5724' ],
+          [ 'CVE', '2020-5723'],
+          [ 'URL', 'https://firmware.grandstream.com/Release_Note_UCM6xxx_1.0.20.22.pdf'],
+          [ 'URL', 'https://raw.githubusercontent.com/tenable/poc/master/grandstream/ucm62xx/dump_http_user_creds.py']
+        ],
+        'DisclosureDate' => '2020-03-30',
+        'DefaultOptions' => {
+          'RPORT' => 8089,
+          'SSL' => true
+        },
+        'Notes' => {
+          'Stability' => [CRASH_SAFE],
+          'SideEffects' => [],
+          'Reliability' => []
+        }
+      )
+    )
+    register_options([
+      OptString.new('TARGETURI', [true, 'Base path', '/']),
+      OptInt.new('ID_SCAN', [true, 'The number of initial user ids to scan', 30])
```

Comment:
Initial user IDs, or total user IDs? Seems like this controls total IDs and is more of a range, so I'm not sure this description  is as helpful to end users in describing what it does as it could be.

Here is a suggestion:

```suggestion
      OptInt.new('ID_SCAN', [true, 'Total number of user IDs to try enumerate for valid users, starting from 1 and going up to and including the number specified', 30])
```